examples/mqtt-client-basic-authentication/mqtt_client_basic_authentication.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on how to configure a MQTT client to use basic authentication.
```

Comment:
```suggestion
description: How to configure an MQTT client to use basic authentication.
```
- Check the format of other BBEs. We do not have to say `BBE on how...` in the description.
- Check if the first letter needs to be capital or not.